documentation/decisions/adr/2023-07-10-wiki-platform.md

```diff
@@ -0,0 +1,349 @@
+# Communications Tooling: Wiki Platform
+
+- **Status:** Accepted
+- **Last Modified:** 2023-07-10 <!-- REQUIRED -->
+- **Related Issue:** [#30](https://github.com/HHS/grants-equity/issues/30) <!-- RECOMMENDED -->
+- **Deciders:** Lucas Brown, Aaron Couch, Billy Daly, Sarah Knopp, Sumi Thaiveettil
+- **Tags:** communucations, open source, wiki
+
+## Context and Problem Statement
+
+The [communications platform milestone](milestone) identifies a series of platforms through which the Grants API project needs to engage both internal and external stakeholders. One of these platforms is a wiki for storing notes, documents, and other content about the project. Ideally we would select a platform that balances ease of use and flexibility with the cost of implementing and maintaining the wiki.
+
+The goal of this ADR is to evaluate a series of potential wiki platforms and determine which one best fits the needs and objectives of this project based on the decision criteria outlined below.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
+
+- **Usability:** Non-technical users should be able to access and create content with minimal training or guidance.
+- **Public Access:** Members of the public should be able to read public documentation in the wiki without needing to sign up or login to a service.
+- **Content Review:** Collaborators should be able to review and edit draft content before those changes are published.
+- **Comments:** Reviewers should be able to leave in-line comments on content that they are reviewing.
```

Comment:
Going back to note that we've discovered that our chosen wiki (GitBook) does not allow inline comments within tables, which is particularly challenging since we use a lot of tables.